code/procs/mobprocs/chatprocs.dm

```diff
@@ -1023,7 +1023,8 @@
 /// -null to give a general system message
 /// -mob to make a mob speak
 /// -flock_structure for a structure message
-/proc/flock_speak(atom/speaker, message as text, datum/flock/flock, speak_as_admin = FALSE)
+/// involuntary overrides the sentient styling for messages generated by the possessed flock critter
+/proc/flock_speak(atom/speaker, message as text, datum/flock/flock, involuntary = FALSE, speak_as_admin = FALSE)
```

Comment:
Is using involuntary needed at all, considering the override is determined by is_npc, which is set by there being a controller?

Reply:
The issue is that there are some messages from a controlled drone that we don't want to highlight, such as the automatic "ow this person hit me" responses.

Reply:
Ok yeah. I feel maybe it's a bit odd for Flockdrones that have a controller to be reporting when attacked, since it's not really the controller that's reporting it, but it's still helpful. If you don't want to remove, was thinking maybe could override say() on Flockdrones and just have the value passed to the parent be related to if there's a controller?